src/layout.rs

```diff
@@ -355,6 +359,27 @@ impl Layout {
         }
     }
 
+    /// Initialize an empty cache with a custom size. The cache is keyed on the layout and area, so
+    /// that subsequent calls with the same parameters are faster. The cache is a LruCache, and
+    /// grows until `cache_size` is reached.
+    ///
+    /// Returns true if the cell's value was set by this call.
+    /// Returns false if the cell's value was not set by this call, this means that another thread
+    /// has set this value or that the cache size is already initialized.
+    ///
+    /// Note that a custom cache size will be set only if this function:
+    /// * is called before [Layout::split()] otherwise, the cache size is `DEFAULT_CACHE_SIZE`.
```

Comment:
The following will link to the const correctly:
```suggestion
    /// * is called before [Layout::split()] otherwise, the cache size is [`Self::DEFAULT_CACHE_SIZE`].
```